backup_and_restore/disaster_recovery/about-disaster-recovery.adoc

```diff
@@ -11,6 +11,11 @@ how to recover from several disaster situations that might occur with their
 more of the following procedures to return your cluster to a working
 state.
 
+[IMPORTANT]
+====
+Disaster recovery requires you to have at least one healthy master host.
```

Comment:
s/master/control-plane?

Reply:
We do use "master host" mostly throughout these docs, so I don't want to introduce it in just this change. We do have efforts planned to s/master/control plane throughout the entire doc set coming down the line